### DISCUSSION TITLE
Making the status square rather than round?

### DISCUSSION THREAD

**Discussion by HiImJulien** [catppuccin/tmux#439] [Help] [answered]
Hey there!
I've just migrated my config to the latest tagged release.
While I am at it, I'd like change corners of my status bar to be square rather than being round.

E.g.:

I want this

![image](https://github.com/user-attachments/assets/aba19bad-2615-4f70-aa13-43802aba4044)

to look like this

![image](https://github.com/user-attachments/assets/9578f8e2-3304-420f-95f9-f1dbc13c7278)

I've been going through the documentation, but I can't seem to find the right option to do so.

Thanks in advance! :)

EDIT:

This is my current bar:

![image](https://github.com/user-attachments/assets/dffd1ed8-04b3-4164-805a-2261adcd2e3d)

I like to keep things quite minimal, so the only theme settings I've set are:

```
set -g @catppuccin_flavor "mocha"
set -ogq @catppuccin_window_status_style "basic"

set -g status-left ""
set -g status-right "#{E:@catppuccin_status_application}"
```

**Comment by tembbo** [comment]
Hi, just remove the `@catppuccin_window_status_style`, or set it to `basic`.

https://github.com/catppuccin/tmux/blob/main/docs/reference/configuration.md#window

**Reply by HiImJulien** [reply]
This option seems to only affect the window bar, but not the status bar :(
Seed the edit for what I mean.

**Reply by tembbo** [reply, marked as answer]
I dont know if `set -g @catppuccin_status_left_separator  ""` and `set -g @catppuccin_status_right_separator` will work, but it worked in old versions

**Reply by HiImJulien** [reply]
Indeed it was the option `@catppuccin_status_left_separator`. Thank you very much!

**Reply by tembbo** [reply]
No problem, glad i could help

**Reply by kjnsn** [reply]
QQ: why are you using `-ogq` to set the option? `-o` means it won't change the value if the option is already set, which doesn't seem like what you want in a config file
